examples/deploy-heroku/README.md

```diff
@@ -49,15 +49,15 @@ heroku ps:scale web=1
 ### 5. Create Slack App
 
 1. Follow the [Getting Started with Bolt for JavaScript][4] guide to:
-    - Create a Slack app
-    - Add required scopes
-    - Subscribe to required events 
+   - Create a Slack app
+   - Add required scopes
+   - Subscribe to required events
 2. Follow the [Deploying to Heroku with Bolt for JavaScript][1] guide to:
-    - Update your **Request URL** for actions and events
+   - Update your **Request URL** for actions and events
 
 ### 6. Bonus: Deploy to Heroku with one click
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/slackapi/bolt-js)
```

Comment:
I think having this button in a subdirectory is causing strange redirects. It's replaced with the redirected result here, which I find to be working well 🙏 

https://devcenter.heroku.com/articles/heroku-button#adding-the-heroku-button

Reply:
This link worked for me 💯

Reply:
🤔 I'm finding that navigating **directly** to the link (that was removed) works, but clicking it from the following pages doesn't:

`main`: https://github.com/slackapi/bolt-js/blob/main/examples/deploy-heroku/README.md
`fork`: https://github.com/zimeg/bolt-js/blob/main/examples/deploy-heroku/README.md